app/channels/run_channel.rb

```diff
@@ -0,0 +1,16 @@
+class RunChannel < ApplicationCable::Channel
+  def subscribed
+    run = Run.find36(params[:run_id])
+    stream_for(run)
+  end
+
+  class << self
```

Comment:
Not sure about class methods, but all public methods are callable from the JS side of things, which means that anyone can all them through the dev console and pass any data that they want to the functions defined here.

Reply:
Tested this and it does not seem that I can call class methods.